### ISSUE TITLE
Similar branches

### ISSUE THREAD

**Issue by lexborisov** [pgpool/pgpool2#52]
Hello, everyone!

Same comparison in branching. Please, see line [773](https://github.com/pgpool/pgpool2/blob/master/src/parser/outfuncs.c#L773) and [780](https://github.com/pgpool/pgpool2/blob/master/src/parser/outfuncs.c#L780) of [src/parser/outfuncs.c](https://github.com/pgpool/pgpool2/blob/master/src/parser/outfuncs.c) file.

```C
if (node->jointype == JOIN_INNER) 
/* and */
else if (node->jointype == JOIN_INNER)
```

```C
	if (node->jointype == JOIN_INNER) 
	{
		if (node->usingClause == NIL && node->quals == NULL && !node->isNatural)
			appendStringInfoString(str, " CROSS JOIN ");
		else
			appendStringInfoString(str, " JOIN ");
	}
	else if (node->jointype == JOIN_INNER)
		appendStringInfoString(str, " JOIN ");
```

Which is correct?

**Comment by pengbo0328** [comment]
@lexborisov Thank you for pointing out this issue.

line [780](https://github.com/pgpool/pgpool2/blob/master/src/parser/outfuncs.c#L780) should be deleted.

I have fixed this issue d34d808bbebb132f1f9e7595c72ee253eef5462f

**Comment by lexborisov** [comment]
Thanks!
